### ISSUE TITLE
How to render positive difference bar of MACD with color red?

### ISSUE THREAD

**Issue by diaodou** [andredumas/techan.js#149]
I found that MACD component doesn't support such feature.
Just like class "path.candle.up" in Candle charts.

**Comment by andredumas** [comment]
That's correct, the difference path does not distinguish between positive/up and negative/down. This would have be a new feature.

**Comment by nosachamos** [comment]
This would be really useful.
